tests/0113-cooperative_rebalance.cpp

```diff
@@ -907,22 +915,28 @@ static void b_subscribe_with_cb_test(rd_bool_t close_consumer) {
   std::string topic_name = Test::mk_topic_name("0113-cooperative_rebalance", 1);
   std::string group_name =
       Test::mk_unique_group_name("0113-cooperative_rebalance");
-  test_create_topic(NULL, topic_name.c_str(), 2, 1);
+  test_create_topic(NULL, topic_name.c_str(), 2, -1);
 
   DefaultRebalanceCb rebalance_cb1;
   RdKafka::KafkaConsumer *c1 = make_consumer(
       "C_1", group_name, "cooperative-sticky", NULL, &rebalance_cb1, 25);
   DefaultRebalanceCb rebalance_cb2;
   RdKafka::KafkaConsumer *c2 = make_consumer(
       "C_2", group_name, "cooperative-sticky", NULL, &rebalance_cb2, 25);
-  test_wait_topic_exists(c1->c_ptr(), topic_name.c_str(), 10 * 1000);
+
+  test_wait_topic_exists(c1->c_ptr(), topic_name.c_str(),
+                         tmout_multip(10 * 1000));
+  sleep_for(5);
 
   Test::subscribe(c1, topic_name);
 
   bool c2_subscribed = false;
   while (true) {
-    Test::poll_once(c1, 500);
-    Test::poll_once(c2, 500);
+    /* Version-specific poll timeouts for cooperative rebalancing */
```

Comment:
Please add the reason for doing this in the comment, instead of stating what's done in the code